jquery.jsticky.js

```diff
@@ -1,98 +1,109 @@
 /* jSticky Plugin
- * =============
- * Author: Andrew Henderson (@AndrewHenderson)
- * Date: 9/7/2012
- * Update: 02/14/2013
- * Website: http://github.com/andrewhenderson/jsticky/
- * Description: A jQuery plugin that keeps select DOM
- * element(s) in view while scrolling the page.
- */
+* =============
+* Author: Andrew Henderson (@AndrewHenderson)
+* Author: Mike Street (@mikestreety)
```

Comment:
Ah thanks - I didn't realise this commit would come over in the PR. Still yet to fully 100% understand how they work!